circle.yml

```diff
@@ -0,0 +1,4 @@
+general:
+  branches:
+    ignore:
+      - update_dockerfile # list of branches to ignore
```

Comment:
I don't think we have this branch on here.. Also, Circle integration would be a separate PR for #11 ?

Reply:
Will remove this file.